pages/report-hazard/script.js

```diff
@@ -1001,3 +1002,12 @@ reportHazardForm.addEventListener('scroll', function (event) {
     continueBtnContainer.style.boxShadow = 'none';
   }
 });
+
+//imprimr currentReport siempre
+function handleHashChange() {
+  var newUrl = window.location.hash;
+  console.log('Nueva URL:', newUrl);
+  console.log(currentReport);
```

Comment:
Remove the console logs

Reply:
Sorry that is just for testing purposes I should have deleted everything after line 1005